web/swoopstyle/html/1-5philosophy1-o-1/index.html

```diff
@@ -0,0 +1,185 @@
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>CodeTap Academy Rocks</title>
+  </head>
+
+  <body>
+    <h2>Philosphy 1-o-1</h2>
+    <br />
+    <br />
+    <!-- 2 How to make a bigger heading -->
+    <pre style="text-align: center"> 
+       <em><img src="plato.jpg" width="250" height="250"></em></pre>
+    <h1 style="font-size: 25px">Plato (c. 427– c. 347 B.C.)</h1>
+
+    <p>
+      The dialogue form in which Plato writes is more than a mere literary
+      device; it is instead an expression of Plato’s understanding of the
+      purpose and nature of philosophy. For Plato, philosophy is a process of
+      constant questioning, and questioning necessarily takes the form of
+      dialogue. Near the end of the Phaedrus, Socrates expresses his
+      reservations about written texts, worrying that people will cease to think
+      for themselves when they have someone else’s thoughts written out in front
+      of them. Plato took it upon himself to write his thoughts down anyway, but
+      he was careful not to write them in such a way that we could easily
+      assimilate his thoughts rather than thinking for ourselves. Many of the
+      dialogues reach no definite conclusions, and those that do generally
+      approach those conclusions by casting doubts and examining possible
+      counterarguments. Plato cannot be there in person to share his thoughts
+      with us, but he wants to ensure that we think through them ourselves.
+      <br />
+      <br />
+      In keeping with this emphasis on dialogue form, Plato develops an
+      increasingly complex conception of dialectic, or logical argument, as the
+      engine that drives philosophical investigations. In the early dialogues,
+      dialectic consists of Socrates cross-examining and refuting his
+      interlocutors until he brings them to a state of perplexity, or aporia.
+      Beginning with the Meno, Plato recognizes that dialectic can lead people
+      not only to recognize their errors but also to positive discoveries, as
+      Socrates does with the slave boy in the Meno. Plato is sufficiently
+      impressed with the possibilities of the dialectic that, in the Republic,
+      he makes it the highest achievement of his rigorous education program. The
+      Phaedrus introduces a more systematic version of the dialectic, seeing it
+      as a matter of “division and generalization,” whereby we analyze concepts
+      so as to understand the precise relations between them. This process of
+      division and generalization becomes increasingly sophisticated throughout
+      Plato’s works, and we witness advanced versions of it in the Parmenides
+      and the Sophist.
+    </p>
+    <a href="https://www.sparknotes.com/philosophy/plato/themes/"
+      >Click here: Plato's themes, arguments & ideas</a
+    >
```

Comment:
It would be nicer like this:
```
    <a href="https://www.sparknotes.com/philosophy/plato/themes/">
      Click here: Plato's themes, arguments & ideas
    </a>
```
to increase readability.

Reply:
👍